docs/howto/garbage-collection/standalone-gc.md

```diff
@@ -72,6 +72,66 @@ docker pull treeverse/lakefs-sgc:<tag>
 
 ## Usage
 
+### Permissions
+To run `lakefs-sgc`, you'll need AWS and LakeFS users, with the following permissions:
+#### AWS
+The minimal required permissions on AWS are:
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "s3:PutObject",
+        "s3:GetObject"
+      ],
+      "Resource": [
+        "arn:aws:s3:::<bucket>/*"
```

Comment:
Too permesive. Should be this, no?
```suggestion
        "arn:aws:s3:::<storage_namespace>/_lakefs/*"
```

Reply:
Can't use only `_lakefs` as it needs access to the `_data` prefix as well.
But you're right that it doesn't need permissions for the entire bucket, only the storage namespace prefix.
Changed accordingly.